### ISSUE TITLE
Cannot read property 'jQuery' of undefined

### ISSUE THREAD

**Issue by JeffWScott** [dmester/jdenticon#35]
```
Uncaught TypeError: Cannot read property 'jQuery' of undefined
    at eval (jdenticon.min.js:5)
    at Object.eval (jdenticon.min.js:9)
    at eval (jdenticon.min.js:235)
    at Object../node_modules/jdenticon/dist/jdenticon.min.js (popup.js:1)
    at n (popup.js:1)
    at eval (Jdenticon.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options:7)
    at Object../node_modules/babel-loader/lib/index.js!./node_modules/vue-loader/lib/index.js?!./src/js/Popup/components/Jdenticon.vue?vue&type=script&lang=js& (popup.js:1)
    at n (popup.js:1)
    at eval (VM66 Jdenticon.vue:2)
    at Module../src/js/Popup/components/Jdenticon.vue?vue&type=script&lang=js& (popup.js:1)
```

Environment:
```
"vue": "^2.5.17",
"jdenticon": "^2.1.1"
```

This is the component:
```
<template>
  <div v-html='identicon' />
</template>

<script>
 import jdenticon from 'jdenticon';

  export default {
      data: () => ({
      }),
      computed: {
        identicon: function() {
          return jdenticon.toSvg('something', 40);
        }
      }
  }
</script>

<style>
</style>
```

Any ideas?  I've googled the crap out of it. I can't seem to get a good solution.

**Comment by dmester** [comment]
Thanks for reporting this issue! Jdenticon has a UMD wrapper to make it possible to use Jdenticon with multiple loaders. It uses the `this` keyword to get hold of the `window` object. This does not work in some setups. I will update the UMD wrapper in the next release, which will hopefully solve your issue.

My intention is to release a new version next week. I will keep you updated.

**Comment by dmester** [comment]
As a workaround you could try to either:

* Load the jdenticon js file in a separate script tag before your Vue bundle, or
* Replace `this` with `self` or `window` in dist/jdenticon.js line 47, and then import this file specifically, instead of the minified file.

**Comment by JeffWScott** [comment]
Hey, thanks for the response!  

BTW I LOVE Jdenticon. Keep up the good work!!

**Comment by dmester** [comment]
Thanks, I'm glad you like it! I just released v 2.2.0 that might fix this error. I tried to reproduce the error with v2.1.1 in an empty application created with @vue/cli, but had no luck. For me your component worked with both v2.1.1 and the new v2.2.0.

Maybe you could try the new version of Jdenticon in your application to see if it solves the issue?
